java_training/src/com/lftechnology/december31/DateTest.java

```diff
@@ -0,0 +1,133 @@
+package com.lftechnology.december31;
+
+/**
+ * Write a test application named DateTest that demonstrates class Date's
+ * capabilities.
+ * 
+ * @author nerisa
+ * 
+ */
+public class DateTest {
+
+	public static void main(String[] args) {
+		DateUtility date = new DateUtility(2, 1, 2015);
+		date.displayDate();
+		date.setDay(3);
+		date.displayDate();
+		date.setMonth(2);
+		date.displayDate();
+		date.setYear(2016);
+		date.displayDate();
+		DateUtility date2 = new DateUtility(-2, 0, 23);
+		date2.displayDate();
+
+	}
+
+}
+
+/**
+ * Create a class called Date that includes three pieces of information as
+ * instance variables a month (type int), a day (type int) and a year (type
+ * int). Your class should have a constructor that initializes the three
+ * instance variables and assumes that the values provided are correct. Provide
+ * a set and a get method for each instance variable. Provide a method
+ * displayDate that displays the month, day and year separated by forward
+ * slashes (/).
+ * 
+ * @author nerisa
+ * 
+ */
+class DateUtility {
+	private int day;
+	private int month;
+	private int year;
+
+	/**
+	 * Sets the instance variables
+	 * 
+	 * @param day
+	 * @param month
+	 * @param year
+	 */
+	public DateUtility(int day, int month, int year) {
+		boolean formatIncorrect = false;
+		if (day < 0 || day > 31) {
```

Comment:
I am February and i find this offensive. Wait I am April, June , September , November too.
I don't know if such criterion is needed now. You could incoporate those too if you want.

But, I appreciate what you have done. Good Work.